openshift/aws-resource-exporter.yaml

```diff
@@ -88,6 +97,16 @@ objects:
       port: 9115
       targetPort: 9115
     type: ClusterIP
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    annotations:
+      qontract.recycle: "true"
+    labels:
+      app: ${NAME}
+    name: ${NAME}
```

Comment:
the name of the configmap should be configurable (think multiple instances in a single namespace)

Reply:
or actually (yes, i'm thinking about this too much), make it configurable (like SECRET_NAME)

Reply:
Added it in now - it has a default after all.